sensu/plugins/check-keystone-expired-tokens.py

```diff
@@ -18,6 +18,15 @@
 import os
 import sys
 
+from datetime import datetime
+
+from keystone.cmd import cli
+from keystone.common import environment
+from keystone import token
+from keystone.common import sql
+from keystone.token.persistence.backends.sql import TokenModel
+from keystone.token.persistence.backends.sql import Token
```

Comment:
I think the ordering is a bit weird, but I see that this was just lifted from another section.